website/src/components/wordpress.tsx

```diff
@@ -69,33 +70,39 @@ export const WordpressPageContents = ({
 
 const parseOptions: HTMLReactParserOptions = {
   replace(node) {
-    const style = /\[(\w*):([0-9]*)-?([0-9]*)?:?(audio)?(join)?\]/ // [DocName:Start(-OptionalEnd):?(audio?)(join?)]
+    const wordEmbedStyle = /\[(\w*):([0-9]*)-?([0-9]*)?:?(audio)?(join)?\]/ // [DocName:Start(-End?):?(audio?)(join?)]
+    const referenceEmbedStyle = /\[(\w*)\]/ // [search | glossary]
```

Comment:
So through this syntax, we can insert a lexical search anywhere using Wordpress? Am I reading this code correctly?